studio/lib/payloads/compensations.ts

```diff
@@ -21,6 +21,13 @@ export interface SalariesPage {
   salaries: string;
 }
 
+export interface BonusesByLocationPage {
+  _type: string;
+  _key: string;
+  location: Reference;
```

Comment:
Jeg har skrevet det slik: location: { _ref: string; _type: string }, men det er kanskje ryddigere å bruke det som et interface?

Reply:
Ja, jeg har definert en global type ettersom dette gjelder for alle referanser.